src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseSource.iOS.cs

```diff
@@ -589,8 +589,11 @@ private CGSize GetTemplateSize(DataTemplate dataTemplate, NSString elementKind,
 				{
 					container.Style = style;
 				}
-
-				container.ContentTemplate = dataTemplate;
+				
+				if (!container.IsContainerFromTemplateRoot)
```

Comment:
@davidjohnoliver were there tests for that specific scenario in ListView?

Reply:
There are tests for the container-as-template-root scenario, but presumably not covering the bug that @dr1rrb encountered